### ISSUE TITLE
Rename 4-tuple routing?

### ISSUE THREAD

**Issue by martinduke** [quicwg/load-balancers#171]
The '11' codepoint might be better defined. It would be good, for instance, if a stateful load balancer actually stored the observed CID so that the connection can survive a NAT rebinding.

**Comment by martinduke** [comment]
Actually this is a bit harder than we'd like.

If the load balancer is stateless, then the short-header processing would have to be based on the destination connection ID to be robust to NAT rebinding. But then, as the connection ID will change mid-handshake, long-header processing needs a different approach. I don't see how, logically, a stateless LB can maintain consistency between these two results unless we force the unconfigured server to retain most of the client-generated CID.

**Comment by martinduke** [comment]
Having reflected on this a bit more, this is what I think we should do:
Keep 0b11 as 4-tuple routing. However Load Balancers MAY observe destination connection IDs and keep state on them, to later survive a NAT rebinding.

I see two use cases for 0b11: 
(1) a general purpose server that always uses QUIC-LB, whether or not it's behind an LB, for simplicity. So when it's not behind a Layer 4 LB it just generates random CIDs except for the first byte. As an added bonus, if there *is* an unknown L4 LB in front of it the behavior will not be pathological, at least until there's a migration. But in general, the rebinding performance is not all that important.
(2) As a panic mode when things are misconfigured or the server has run out of nonces. Again, I'm not sure rebinding performance is super-critical.

In other words, if a server is behind an L4 load balancer is and is using 0b11, something has gone very wrong.
